web/client/plugins/Dashboard.jsx

```diff
@@ -28,10 +28,11 @@ const WidgetsView = compose(
             getWidgetsDependenciesGroups,
             showConnectionsSelector,
             isDashboardLoading,
-            (resource, widgets, layouts, dependencies, selectionActive, editingWidget, groups, showGroupColor, loading) => ({
+            isBrowserMobile,
+            (resource, widgets, layouts, dependencies, selectionActive, editingWidget, groups, showGroupColor, loading, isMobile) => ({
                 resource,
                 loading,
-                canEdit: (resource ? !!resource.canEdit : true),
+                canEdit: isMobile ? !isMobile : resource && !!resource.canEdit,
```

Comment:
Here if resource is not present, the result should be true. This is because new dashboard doesn't have resource yet, and the new context have to be editable. 
Please add a test for the plugin if possible, with resource test and verify this condition

Reply:
Yes. It works as expected. And have added test cases to verify the condition